### ISSUE TITLE
DNS server IPv6 support

### ISSUE THREAD

**Issue by hannesm** [mirage/ocaml-dns#253]
The server implementation now accept a Mirage_stack.V4V6, and thus listen on IPv4 and IPv6 addresses. What is left to do is further code reading and review for notifications (should be sent to each NS only once - IPv4 or IPv6 address), DNS TSIG keys (embedding IPv4 addresses at the moment for the notification and knowledge which server to contact).

since multiple / duplicated notify packets are just ignored, it is safe to postpone this to a later point.

**Comment by hannesm** [comment]
with the rise of [happy-eyeballs](https://github.com/roburio/happy-eyeballs), this can as well be used to notify name servers (same name, both A and AAAA records).
